syft/frameworks/torch/mpc/securenn.py

```diff
@@ -288,6 +290,19 @@ def msb(a_sh):
     else:
         return a
 
+# Calculates how many times the shares of a will wrap around L
+# a SHOULD BE A PUBLIC VALUE
+def wrap(a_sh, L):
+    # The code could be simpler, but then there are some overflow errors
+    total = torch.zeros(a_sh.shape)
+    remainders = torch.zeros(a_sh.shape)
+    for w in a_sh.locations:
+        cur = a_sh.child[w.id].copy().get()
```

Comment:
Don't call get() but move() instead, and move to the first worker for example. The local worker should see this values as you commented, but the other workers can: it's a public value that they all own.

Reply:
Where should I add the test?

Reply:
Add a test close to the one for secure_nn (I think it's the comparison tests in precision.py or additive_sharing.py)
Where you put more than 2 share holders (maybe 3 and 4 is fine)

Reply:
Looking at 'test_additive_shared.py', the tests have the form 'def test_stuff(workers): alice, bob = (workers['alice'], workers['bob']).
Should I add new tests without these assumptions or change the existing ones? Also, where are these tests called?

Reply:
As you want, take the cleanest option :)

To run you tests:
`pytest <path>/test_additive_shared.py`

Reply:
Done! Do you see anything else to add/fix, or can we proceed with the merge?